### DISCUSSION TITLE
[@types/google-apps-script] Missing sheet.getLastColumn()

### DISCUSSION THREAD

**Discussion by diamond-fish** [DefinitelyTyped/DefinitelyTyped#66404] [Issues with a @types package]
I just noticed `sheet.getLastRow()` exists, but not `sheet.getLastColumn()`.

[a comment by an automated account is not shown]

**Comment by mtgto** [comment]
It seems to already exists. Can you show me your sample code?

- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbbcca72cca2a6d850eba764af506b4867a90592/types/google-apps-script/google-apps-script.spreadsheet.d.ts#L1720
- https://developers.google.com/apps-script/reference/spreadsheet/sheet#getLastColumn()

**Comment by diamond-fish** [comment]
I should have recorded this issue when it happened because now I cannot reproduce it 😅. Thank you for checking.
